irida_staramr_results/parser.py

```diff
@@ -0,0 +1,35 @@
+import yaml
+import logging
+import sys
+
+
+def parse_config(config_file_path):
+    """
+    Parse configuration YAML file provided by the user.
+    It assumes the file name to be "example-config.yml".
```

Comment:
Is this still true that it expects the file name to be `example-config.yml`? I thought we were entering that ourselves via an argument.

Reply:
Right. Good catch. I missed this. I removed it now.